tests/accesscontrol/tests/access_control_pod_cluster_role_bindings.go

```diff
@@ -0,0 +1,77 @@
+package accesscontrol
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	tshelper "github.com/test-network-function/cnfcert-tests-verification/tests/accesscontrol/helper"
+	tsparams "github.com/test-network-function/cnfcert-tests-verification/tests/accesscontrol/parameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/execute"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+)
+
+var _ = Describe("Access-control pod cluster role binding,", func() {
+
+	execute.BeforeAll(func() {
+		By("Define tnf config file")
+		err := globalhelper.DefineTnfConfig(
+			[]string{tsparams.TestAccessControlNameSpace},
+			[]string{tsparams.TestPodLabel},
+			[]string{},
+			[]string{})
+		Expect(err).ToNot(HaveOccurred(), "error defining tnf config file")
+
+	})
+
+	BeforeEach(func() {
+		By("Clean namespace before each test")
+		err := namespaces.Clean(tsparams.TestAccessControlNameSpace, globalhelper.APIClient)
+		Expect(err).ToNot(HaveOccurred())
+
+	})
+	// 56427
+	It("one deployment, one pod, does not have cluster role binding", func() {
+		By("Define deployment  that dont have rolebinfing")
+		dep, err := tshelper.DefineDeployment(1, 1, "accesscontroldeployment")
+		Expect(err).ToNot(HaveOccurred())
+
+		err = globalhelper.CreateAndWaitUntilDeploymentIsReady(dep, tsparams.Timeout)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start test")
+		err = globalhelper.LaunchTests(
+			tsparams.TestCaseNameAccessControlCluterRoleBindings,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(
+			tsparams.TestCaseNameAccessControlCluterRoleBindings,
+			globalparameters.TestCasePassed)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("one deployment, one pod, does  have cluster role binding [negative]", func() {
```

Comment:
In testing I'm getting this error in this test case:
`STEP: Define deployment with cluster role binding  @ 06/24/23 00:55:09.806
  [FAILED] in [It] - /root/ejohnson/syncs/cnfcert-tests-verification/tests/globalhelper/deployment.go:55 @ 06/24/23 01:00:09.835
• [FAILED] [301.641 seconds]
Access-control pod cluster role binding, [It] one deployment, one pod, does  have cluster role binding [negative]
/root/ejohnson/syncs/cnfcert-tests-verification/tests/accesscontrol/tests/access_control_pod_cluster_role_bindings.go:56

  [FAILED] Timed out after 300.004s.
  Deployment is not ready %!(EXTRA string=2023-06-24 00:55:09.831095675 +0300 IDT m=+508.258548028 &DeploymentStatus{ObservedGeneration:0,Replicas:0,UpdatedReplicas:0,AvailableReplicas:0,UnavailableReplicas:0,Conditions:[]DeploymentCondition{},ReadyReplicas:0,CollisionCount:nil,})
  Expected
      <bool>: false
  to equal
      <bool>: true
  In [It] at: /root/ejohnson/syncs/cnfcert-tests-verification/tests/globalhelper/deployment.go:55`

Reply:
can u check now its need to work